recipes-qt/qt-kiosk-browser/qt-kiosk-browser_git.bb

```diff
@@ -0,0 +1,32 @@
+SUMMARY = "Qt Kiosk Browser"
+DESCRIPTION = "Provides a simple but highly configurable browser for use on Kiosk devices"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1ebbd3e34237af26da5dc08a4e440464"
+LICENSE = "GPLv3"
+
+DEPENDS = "qtwebengine"
+
+SRC_URI = " \
+    git://github.com/OSSystems/qt-kiosk-browser;protocol=https \
+    file://${PN}.conf \
```

Comment:
QA Issue: qt-kiosk-browser: SRC_URI uses PN not BPN [src-uri-bad]

Will fix in master-next.